test/server/routes/auth.test.js

```diff
@@ -406,6 +407,37 @@ describe('AUTH', () => {
           done();
         });
     });
-
+  });
+  describe('Fetch Books router', () => {
+    it('It should get book with query string page and limit', (done) => {
+      chai.request(server)
+        .get(`${BOOKS_BASE_URL}?page=1&limit=10`)
+        .end((error, response) => {
+          expect(response).to.have.status(200);
+          expect(response.body).to.be.an('object');
+          expect(response.body.message).to.equal('request successful');
+          done();
+        });
+    });
+    it('It should get book without query string page and limit', (done) => {
+      chai.request(server)
+        .get(`${BOOKS_BASE_URL}`)
+        .end((error, response) => {
+          expect(response).to.have.status(200);
+          expect(response.body).to.be.an('object');
+          expect(response.body.message).to.equal('request successful');
+          done();
+        });
+    });
+    it('It should get response:"no more books found" if current page is greater than pages', (done) => {
+      chai.request(server)
+        .get(`${BOOKS_BASE_URL}?page=10000000&limit=10`)
+        .end((error, response) => {
+          expect(response).to.have.status(200);
+          expect(response.body).to.be.an('object');
+          expect(response.body.message).to.equal('no more books');
+          done();
+        });
+    });
```

Comment:
There are no test cases for when a user tries to input wrong data types e.g alphabets or special characters in the parameter field for page or limit.

Reply:
done